src/components/sortControl/sortControl.tsx

```diff
@@ -0,0 +1,34 @@
+import './sortControl.css';
+
+interface SortControlProps {
+  sortOptions: movieSortOption[];
```

Comment:
This approach makes the component more flexible what is redundant in our case. Please simplify props: just selected option and callback.